tests/commands/config.py

```diff
@@ -0,0 +1,785 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import json
+
+import pytest
+
+from django.core.management import call_command
+from django.core.management.base import CommandError
+
+from pootle.core.delegate import config
+from pootle_project.models import Project
+
+
+def _repr_value(value):
+    if not isinstance(value, (str, unicode)):
+        value_class = type(value).__name__
+        return (
+            "%s(%s)"
+            % (value_class,
+               json.dumps(value)))
+    return value
+
+
+def _test_config_get(out, key, model=None, instance=None, asjson=False):
```

Comment:
not sure i follow. The command has 2 modes string mode and json mode, this lets the test flip between